src/common/hobject.h

```diff
@@ -250,7 +250,8 @@ struct hobject_t {
   }
   void set_bitwise_key_u32(uint32_t value) {
     hash = _reverse_bits(value);
-    build_hash_cache();
```

Comment:
Please add a comment to this and build_hash_cache() to ensure that we remember to update both.